### PR TITLE
Add more debug info to openapi generator

### PR DESCRIPTION
**Pull request by gmarek**
@mbohlool PTAL

cc @sttts

### CODE REVIEW

File: pkg/generators/openapi.go [hunk 1]
```diff
@@ -328,6 +328,7 @@ func (g openAPITypeWriter) generateMembers(t *types.Type, required []string) ([]
 			required = append(required, name)
 		}
 		if err = g.generateProperty(&m, t); err != nil {
+			glog.Errorf("Error when generating: %v, %v\n", name, m)
```
Comment (sttts) [inline pkg/generators/openapi.go]: isn't the error printed by the caller of `generateMembers`? Or could it? Printing and returning the same error always looks odd.

Comment (gmarek) [inline pkg/generators/openapi.go]: Returned error doesn't contain information on for what it failed. We can add it there, but this was just simpler.

Comment (sttts) [inline pkg/generators/openapi.go]: As it's only the generator, I am fine with this.

**Comment by mbohlool** [comment]
LGTM.

**Pull request merged**